py3-changelog-chug.yaml

```diff
@@ -0,0 +1,79 @@
+package:
+  name: py3-changelog-chug
+  version: 0.0.3
+  epoch: 0
+  description: Parser library for project Change Log documents.
+  copyright:
+    - license: AGPL-3.0-only
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: changelog-chug
+  import: chug
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
+
+environment:
+  contents:
+    packages:
+      - py3-supported-build-base
+      - py3-supported-docutils
+      - py3-supported-semver
```

Comment:
Are these actually needed at buildtime and runtime?

Reply:
Yes, this is required at build time and runtime both. 

```bash
(hacky_ckHyws) ~/t/hacky_ckHyws $ uv pip show changelog-chug
Name: changelog-chug
Version: 0.0.3
Location: /home/user/tmp/hacky_ckHyws/.venv/lib/python3.12/site-packages
Requires: docutils, semver
Required-by:
(hacky_ckHyws) ~/t/hacky_ckHyws $ uv pip tree
changelog-chug v0.0.3
├── docutils v0.21.2
└── semver v3.0.2
```
but please let me know if there's a better way to determine this? As of now, I mostly rely on logs `ModuleNotFound` error messages and pip output.

Reply:
pip is the best way to go about it